### DISCUSSION TITLE
Make Turborepo website performant

### DISCUSSION THREAD

**Discussion by sguergachi** [vercel/turborepo#8052] [Ideas]
### Goals

1. Represent the goals of turborepo/turbopack with a high performance site
2.
3.

### Non-goals

1.
2.
3.

### Background

When scrolling through the turborepo/turborepo homepage I noticed it was very laggy. This is a turnoff for someone interested in a high performance library, because a poor performing website doesn't give confidence the library will be performant, it leaves a bad impression.

https://github.com/vercel/turbo/assets/615386/2508407a-5d7c-496c-a92a-fe7fb0a2ae69

I am running Chrome on Windows.

### Proposal

Make the home page very fast to scroll, load, etc.

**Comment by sguergachi** [comment]
I used lagradar to better indicate lag

https://github.com/vercel/turbo/assets/615386/dbaf951f-1e13-4c93-886b-bf4c5f119255

**Comment by versecafe** [comment]
@sguergachi The docs site no longer seems to have public source code just mdx files but based off when it was still accessible the following fixes can be made, swap `import { motion } from "framer-motion";` to `import { m } from "framer-motion";`, move the landing pages over to the app router, and make sure there aren't isn't any unused css still sitting around in the various `*.module.css` files anymore.

If I'm wrong and the site source is still up I can go off and fix these quickly and likely find a few other improvements that can be made but the best I can do now is point out issues that used to exist.

**Comment by versecafe** [comment]
@anthonyshew Are the docs open source somewhere still so I can go fix these issues?
